src/modules/admin/components/users/UserAccessHistory.tsx

```diff
@@ -88,7 +88,6 @@ const UserAccessHistory = ({ accessEntityType }: Props) => {
             label='Search client access'
             value={search}
             onChange={setSearch}
-            helperText={null}
```

Comment:
Removed this helperText from `ClientTextSearchInput` and moved it one level up into `ClientTextSearchForm` because of this conversation: https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL?node-id=1650:25045&mode=design#700990401